backend/main.py

```
@@ -170,9 +201,18 @@
     except docker.errors.BuildError as e:
         logging.error(f"Docker build failed: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)